packages/core/types/src/modules/documents/params/attributes/relations.ts

```diff
@@ -0,0 +1,46 @@
+import type * as Schema from '../../../../schema';
+
+import type * as UID from '../../../../uid';
+import type { If } from '../../../../utils';
+
+import type { ID, DocumentID } from './id';
+
+type ShortHand = ID;
+type LongHandEntity = { id: ID };
+type LongHandDocument = { documentId: DocumentID; locale?: string };
+type LongHand = LongHandEntity | LongHandDocument;
+
+interface PositionalArguments {
+  before?: ID;
+  after?: ID;
+  start?: boolean;
+  end?: boolean;
+}
+
+type WithPositionArguments<T> = T & { position?: PositionalArguments };
+
+type Set = { set: ShortHand[] | LongHand[] | null };
+type Connect = { connect: ShortHand[] | WithPositionArguments<LongHand>[] };
+type Disconnect = { disconnect: ShortHand[] | LongHand[] };
+
+type FullUpdate = Set;
+type PartialUpdate = Partial<Connect & Disconnect>;
```

Comment:
I believe this is a copy of what we have in the entity-service types too, just so I know for the future, is the plan to maintain both at the same time? or we plan to unify them?

Reply:
Yep it is, the plan is to only maintain the DS one, the ES is deprecated and won't receive updates.